### ISSUE TITLE
Chat Tab Somehow Not Accessible with Screen Reader

### ISSUE THREAD

**Issue by lbk2907** [BearWare/TeamTalk5#1885]
### Description

I not sure how it happend, but somehow the chat tab will not accessible with screen reader. That's mean I cannot read the channel message if someone send it. I not sure are the sited people able to read the chat. It start happening since TeamTalk version 5.13 Build 120.

### Application

- [ ] qtTeamTalk
- [X] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [ ] Windows
- [ ] macOS
- [X] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

It should accessible with screen reader.

### Actual behavior

Chat tab somehow not accessible with screen reader.

### Steps to reproduce problem

I not really sure how to reproduce it. But one of the way is go to someone name and toggle the subscription switch like for example subscribe to your own voice.

**Comment by beqabeqa473** [comment]
cannot reproduce

On 5/18/23, lbk2907 ***@***.***> wrote:
> ### Description
>
> I not sure how it happend, but somehow the chat tab will not accessible with
> screen reader. That's mean I cannot read the channel message if someone send
> it. I not sure are the sited people able to read the chat. It start
> happening since TeamTalk version 5.13 Build 120.
>
> ### Application
>
> - [ ] qtTeamTalk
> - [X] TeamTalkAndroid
> - [ ] iTeamTalk
> - [ ] TeamTalkClassic
> - [ ] TeamTalkServer
>
> ### Platform
>
> - [ ] Windows
> - [ ] macOS
> - [X] Android
> - [ ] iOS
> - [ ] Linux
>
> ### Expected behavior
>
> It should accessible with screen reader.
>
> ### Actual behavior
>
> Chat tab somehow not accessible with screen reader.
>
> ### Steps to reproduce problem
>
> I not really sure how to reproduce it. But one of the way is go to someone
> name and toggle the subscription switch like for example subscribe to your
> own voice.
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/BearWare/TeamTalk5/issues/1885
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili

**Comment by lbk2907** [comment]
I not sure but it happened on me here. Running on Android 13 and 14. One on Redmi Note 10 Pro with PixelExperience custom rom and one on Pixel 7 Pro. My friend also experience this. I not sure what phone he’s using but he is running on Android 9.

**Comment by amirmahdifard** [comment]
your steps t9 reproduce is not correct

**Comment by lbk2907** [comment]
Like what I said, I don’t know how to exactly reproduce the bug but on my side is it happened when I toggle the subscription switch on one of the user like for example when I subscribe to my own voice.

**Comment by lbk2907** [comment]
Correct me then.

**Comment by beqabeqa473** [comment]
Fixed
